src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -292,7 +292,9 @@ public void ProcessValidationErrors(IEnumerable<IdentityError> errors, User user
                     case "InvalidEmail":
                         reportError("Email", S["Email '{0}' is invalid.", user.Email]);
                         break;
-
+                    case "InvalidToken":
+                        reportError(nameof(user.ResetToken), S["Reset token is invalid."]);
```

Comment:
the key here should probably by `string.Empty` not ResetToken since that isn't a field.
```suggestion
                        reportError(string.Empty, S["The reset token is invalid. Please request a new token."]);
```